internal/namespaces/mnq/v1beta1/custom_nats_test.go

```diff
@@ -0,0 +1,80 @@
+package mnq
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/scaleway/scaleway-cli/v2/internal/core"
+)
+
+func Test_CreateContext(t *testing.T) {
+	t.Run("Simple", core.Test(&core.TestConfig{
+		Commands:   GetCommands(),
+		BeforeFunc: createNATSAccount("NATS"),
+		Cmd:        "scw mnq nats create-context nats-account-id={{ .NATS.ID }}",
+		Check: core.TestCheckCombine(
+			//core.TestCheckGolden(), > As the filenames change, the golden tests do not pass.
```

Comment:
Try to fix the name to have the golden identical

Reply:
core provides `core.TestCheckGoldenAndReplacePatterns` to check goldens with regex to circumvent this kind of issue